docs/classes/embed_EventBus.EventBus.md

```diff
@@ -93,7 +104,7 @@ ___
 
 #### Defined in
 
-embed/EventBus.ts:9
+[embed/EventBus.ts:9](https://github.com/vbrick/rev-sdk-js/blob/c8dd2aa/src/embed/EventBus.ts#L9)
```

Comment:
Why is this hardcoded when the previous version was not?

Reply:
When the script runs within a folder that is part of a github repository it creates links to the source.  When there is no repository it leaves it unlinked.

Reply:
I think its ok. This link is specifically intended to be a github page, so the user can find where things are defined in source.